qtav/qtav.2016-03-02.manifest

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://inqlude.org/schema/release-manifest-v1#",
+  "name": "qtav",
+  "release_date": "2016-03-02",
+  "version": "1.10.0",
+  "summary": "A cross-platform multimedia playback framework based on Qt and FFmpeg.",
+  "urls": {
+    "homepage": "http://www.qtav.org",
+    "vcs": "https://github.com/wang-bin/QtAV/tree/v1.10.0",
+    "download": "http://www.qtav.org/install.html",
+    "announcement": "http://www.qtav.org/blog/1.10.0.html",
+    "custom": {
+        "Wiki": "https://github.com/wang-bin/QtAV/wiki/_pages"
```

Comment:
I would rather link to the entry point of the wiki at https://github.com/wang-bin/QtAV/wiki than the list of pages. This is a more user-friendly way to start.